src/styles/Pay.css

```diff
@@ -0,0 +1,25 @@
+/* #payment-info-table {
+  width: 100%;
+} */
```

Comment:
@bill-feng Let's remove this comment out :)

Reply:
Fixed!